openerp/addons/base/tests/test_orm.py

```diff
@@ -166,6 +166,22 @@ def read_group(interval, domain=None):
                         ['date'], ['date:month', 'date:day'], lazy=False)
         self.assertEqual(len(rg), len(all_partners))
 
+    def test_m2m_store_trigger(self):
+        group_user = self.env.ref('base.group_user')
+
+        user = self.env['res.users'].create({
+            'name': 'test',
+            'login': 'test_m2m_store_trigger',
+            'groups_id': [(6, 0, [])],
+        })
+        self.assertTrue(user.share)
+
+        group_user.write({'users': [(4, user.id)]})
+        self.assertFalse(user.share)
```

Comment:
Cool, that's what I was looking for, thanks! If there was no test for this behavior we can keep this one :-)